.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{

```

Comment:
Is this change necessary? It comes from VSCode settings right?

Reply:
No, it's just my personal configuration for a spell checker. I accidentally pushed it

Reply:
I removed it